BraveWallet/Crypto/Transaction Confirmations/EditGasFeeView.swift

```diff
@@ -0,0 +1,122 @@
+// Copyright 2021 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import SwiftUI
+import BraveCore
+import BraveUI
+import struct Shared.Strings
+
+struct EditGasFeeView: View {
+  // var estimate: BraveWallet.GasEstimation1559
+  
+  private enum GasFeeKind: Hashable {
+    case low, optimal, high, custom
+  }
+  
+  @State private var gasFeeKind: GasFeeKind = .optimal
+  
+  var body: some View {
+    List {
+      Section(
+        header: Text(Strings.Wallet.gasFeeDisclaimer)
+          .foregroundColor(Color(.secondaryBraveLabel))
+          .font(.footnote)
+          .resetListHeaderStyle()
+          .padding(.vertical)
+      ) {
+        Picker(selection: $gasFeeKind) {
+          Text(Strings.Wallet.gasFeePredefinedLimitLow).tag(GasFeeKind.low)
+          Text(Strings.Wallet.gasFeePredefinedLimitOptimal).tag(GasFeeKind.optimal)
+          Text(Strings.Wallet.gasFeePredefinedLimitHigh).tag(GasFeeKind.high)
+          Text(Strings.Wallet.gasFeeCustomOption).tag(GasFeeKind.custom)
+        } label: {
+          EmptyView()
+        }
+        .accentColor(Color(.braveBlurpleTint))
+        .pickerStyle(.inline)
+        .foregroundColor(Color(.braveLabel))
+      }
+      .listRowBackground(Color(.secondaryBraveGroupedBackground))
+      if gasFeeKind == .custom {
+        Section(
+          header: VStack {
+            HStack {
+              Text(Strings.Wallet.gasCurrentBaseFee)
+              Spacer()
+              Text("150 Gwei")
```

Comment:
make verbatim, why this hardcoded value, is it supposed to be a placeholder?

Reply:
Will be replaced, these screens were not yet integrated